src/main/java/model/Race.java

```diff
@@ -0,0 +1,80 @@
+package model;
+
+import java.util.ArrayList;
+import java.util.stream.Collectors;
+import service.MoveStrategy;
+
+/**
+ * Represents a car racing game where multiple cars compete.
+ * Manages the movement and position of cars and determines winners.
+ */
+public class Race {
+    private final ArrayList<Car> cars;
```

Comment:
```suggestion
    private final List<Car> cars;
```
이렇게 업캐스팅을 하게 되면 의존도가 낮아져서 좋다고 합니다!

참고 : https://like-while.tistory.com/7